src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -98,6 +98,14 @@ const getId = ({ pageType, service, variant, env }: GetIdProps) => {
     case LIVE_PAGE:
```

Comment:
Note from huddle: do we want this to have its own path matching code here? /service/live/serviceId should move to the LIVE_PAGE case. Then we don't blur the lines between the different page types.

Reply:
Done